dali/test/python/test_operator_affine_transforms.py

```diff
@@ -92,33 +95,35 @@ def scale_affine_mat(scale, center = None):
 
     return affine_mat
 
-def check_transform_scale_op(scale, center=None, has_input = False, reverse_order=False, batch_size=1, num_threads=4, device_id=0):
-    ndim = len(scale)
+def check_transform_scale_op(scale, center=None, has_input = False, reverse_order=False, ndim=None, batch_size=1, num_threads=4, device_id=0):
+    if ndim is None:
+        ndim = len(scale)
     assert center is None or len(center) == ndim
 
     pipe = Pipeline(batch_size=batch_size, num_threads=num_threads, device_id=device_id, seed = 1234)
     with pipe:
         if has_input:
             T0 = fn.uniform(range=(-1, 1), shape=(ndim, ndim+1))
-            T1 = fn.transforms.scale(T0, device='cpu', scale=scale, center=center, reverse_order=reverse_order)
+            T1 = fn.transforms.scale(T0, device='cpu', scale=scale, center=center, ndim=ndim, reverse_order=reverse_order)
```

Comment:
Can you add a case when `ndim` is not provided at all?

Reply:
That's already most of the tests. Providing `None` is equivalent to not providing anything from the point of view of the operator